idea_town/frontend/templates/idea_town/frontend/index.html

```diff
@@ -1,13 +1,49 @@
-{% extends "idea_town/base.html" %}
+<!doctype html>
+<html>
+<head>
+    <title>Idea Town</title>
+    <link rel="stylesheet" href="{{ static('styles/main.css') }}">
+</head>
+<body>
+  <header><a href="/accounts/login/?next=/">log in</a></header>
+  <!-- container div -->
+  <div id="container">
+    <header>
+      <img id="logo">
+      <div id="avatar-or-signin-button"></div>
+    </header>
+    <div id="hero-space">
+      <!-- not idea town user? show idea town sales pitch.
+           idea town user? show something exciting and fun -->
+      <h1>Idea Town is the Best</h1>
+    </div>
+    <div id="experiments"></div>
+  </div>
+  <footer>
+    <ul>
+      <li><a href="//mozilla.com"><img src="mozilla-logo.png"></a>
+      <li><a href="//blog.mozilla.org">Blog</a>
+      <li><a href="//mozilla.org/privacy">Privacy Policy</a>
+      <li><a href="//mozilla.org/cookies">Cookies</a>
```

Comment:
Ref: #67

I think this link should be `//mozilla.org/privacy/websites/#cookies` (per the [**idea-town-splash**](https://github.com/mozilla/idea-town-splash/blob/6bb0ca832896249240c76fb745ef143c1d446387/routes/data.json#L43-L46) repo).